jinja2/qfdmo/formulaire.html

```diff
@@ -18,6 +18,19 @@
 {% endblock form_modals %}
 
 {% block content_footer %}
+    <div
+        class="qf-px-2w sm:qf-px-0 qf-duration-300 sm:qf-h-fit sm:qf-mr-1w qf-flex qf-flex-row-reverse
+        group-data-[search-form-visible]:qf-hidden"
```

Comment:
Utilisation de https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state 


Je suis parti de l'approche suivante:
- on définit l'état local en javascript (formulaire ouvert ou fermé) 
- on attache l'état local au contrôleur pour pouvoir facilement déclencher d'autres styles éventuels basés sur cet état local
- on stylise en fonction du html pour éviter de manipuler trop de classes tailwind dans le javascript